CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### PaymentSheet
+* [ADDED][8607](https://github.com/stripe/stripe-android/pull/8607) Add builder pattern for PaymentSheet and PaymentSheet.FlowController
```

Comment:
We should update all of our docs for this too. This should be our new preferred integration shape.

Reply:
[Ticket](https://jira.corp.stripe.com/browse/MOBILESDK-2228)